src/Arachne.cc

```diff
@@ -811,6 +802,12 @@ parseOptions(int* argcp, const char** argv) {
             case 'd':
                 disableLoadEstimation = true;
                 break;
+            case 'a':
+                useCoreArbiter = atoi(optionArgument);
```

Comment:
It is not clear what the user is supposed to pass here. I think we had discussed passing in strings, but this does not parse strings.